docs/user-guide/server.rst

```diff
@@ -30,10 +30,11 @@ For a recap of Pulp components and the work they are responsible for, read :ref:
   available workers. If a worker with the same name is started again after being missing, it is
   added into the pool of workers as any worker starting up normally would.
 
-* If ``pulp_celerybeat`` dies, and in case then new workers start, they won't be given work. If
-  existing workers stop, Pulp will continue assigning them work. Once restarted, pulp_celerybeat
-  will synchronize with the current state of all workers. Scheduled tasks will not run while
-  pulp_celerybeat is down, but they will instead run when celerybeat is restarted.
+* If all instances of ``pulp_celerybeat`` die, and new workers start, they won't
+  be given work. If existing workers stop, Pulp will continue assigning them work. Once
```

Comment:
consider rewriting this line to be:

`be given work, or if existing workers stop, Pulp will incorrectly continue assigning them work`